datasets/wise-neowiser.yaml

```diff
@@ -0,0 +1,29 @@
+Name: NEOWISE Reactivation Data | Near-Earth Object Wide-field Infrared Survey Explorer (NEOWISE)
+Description: The NEOWISE dataset contains the 3.4 and 4.6 micron (W1 and W2) single-exposure images and extracted source information from the NEOWISE Reactivation (NEOWISE-R) Mission which started 13 December 2013. NEOWISE scans the sky twice per year with twelve or more independent exposures made on each point on the sky during each survey epoch. Therefore, the NEOWISE archive is a time-domain resource for extracting multiple thermal flux and position measurements of solar system small bodies, as well as background stars and galaxies.
+Documentation: https://irsa.ipac.caltech.edu/Missions/wise.html
+Contact: https://irsa.ipac.caltech.edu/docs/help_desk.html
+ManagedBy: "NASA/IPAC Infrared Science Archive ([IRSA](https://irsa.ipac.caltech.edu)) at Caltech"
+UpdateFrequency: The NEOWISE dataset is updated annually. The data may also be presented in new ways as the products become available.
+Tags:
+  - astronomy
+  - survey
+  - imaging
+  - object detection
+  - parquet
+License: https://irsa.ipac.caltech.edu/data_use_terms.html
+Citation: "If you use the NEOWISE-R Single-exposure Image Sets, please cite the Digital Object Identifier (DOI): https://www.ipac.caltech.edu/doi/irsa/10.26131/IRSA147. If you use the NEOWISE-R Single-exposure Source Database, please cite the Digital Object Identifier (DOI): https://www.ipac.caltech.edu/doi/irsa/10.26131/IRSA144. In addition, follow the [NEOWISE acknowledgement guidelines](https://wise2.ipac.caltech.edu/docs/release/neowise/expsup/sec1_6b.html) and the [IRSA acknowledgement guidelines](https://irsa.ipac.caltech.edu/ack.html)."
+Resources:
+  - Description: The Single-exposure Image Sets consist of more than 20 million calibrated 1016x1016 pix @2.75"/pix FITS image sets for the individual 7.7 sec W1 and W2 NEOWISE survey exposures. Each image set consists of two intensity images, noise maps, and bit-masks indicating pixel use status, one each for the W1 and W2 bands. The Single-exposure Source Database contains time-tagged fluxes, positions and measurement information for more than 150 billion source detections extracted from the Single-exposure images. It is available in Apache Parquet format, partitioned by HEALPix order 5. Additional presentations of this data may be added when available.
```

Comment:
The Single-exposure Source Database (parquet catalogs for years 1-9) is not in the bucket yet. The files have been created and verified but the ancillary files need to be created before uploading. I'm not sure how we feel about submitting the yaml before uploading those.

Reply:
I suppose the Parquet files are so close that it won't cause much harm to submit the yaml file now.